kubernetes/kubapply

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ ! -f "Vars" ]; then
+   echo "File Vars does not exist. Create A file Vars named Vars containing the variables."
+   exit
+fi
+
+FILE=${1}
+
+if [ ! $FILE ]; then
+   echo "Please Provide a file"
+   exit
+fi
+
+
+if [ ! -f $FILE ]; then
+   echo "File $FILE does not exist"
+   exit
+fi
+
+
+COM="cat $FILE | sed \""
```

Comment:
Can the variable "COM" have a more descriptive name ?
As we discussed on Gitter yesterday.